packages/fa-icon-chooser/src/utils/utils.ts

```diff
@@ -3,8 +3,8 @@ import { valid as validSemver } from 'semver';
 import { cloneDeep, get, set } from 'lodash';
 
 const FREE_CDN_URL = 'https://use.fontawesome.com';
-const PRO_KIT_ASSET_URL = 'https://ka-p.fontawesome.com';
-const FREE_KIT_ASSET_URL = 'https://ka-f.fontawesome.com';
+const PRO_KIT_ASSET_URL = 'https://ka-p-epic.fontawesome.com';
+const FREE_KIT_ASSET_URL = 'https://ka-f-epic.fontawesome.com';
```

Comment:
understood this will need to be changed back.  Right now we need for testing.
Once approved, I will remove `-epic`